docker_images/sklearn/app/pipelines/tabular_classification.py

```diff
@@ -27,4 +43,7 @@ def __call__(
         """
         column_values = list(inputs["data"].values())
         rows = list(zip(*column_values))
-        return self.model.predict(rows).tolist()
+        # TODO: we should do sth with the warnings.
+        with warnings.catch_warnings():
+            warnings.filterwarnings("ignore")
+            return self.model.predict(rows).tolist()
```

Comment:
If we don't catch these warnings here, there will always be a 500 response if the user has used a pandas DataFrame to train their model. The new versions of `sklearn` store the feature names used during `fit` and expect the same when `predict` is called, otherwise raise a warning. I don't want to fix those issues here in this PR, rather get something running and then fix the remaining issues in the coming PRs if that's okay with you.